ingestr/src/sources.py

```diff
@@ -1455,48 +1455,45 @@ def handles_incrementality(self) -> bool:
         return True
 
     def dlt_source(self, uri: str, table: str, **kwargs):
-        parse_uri = urlparse(uri)
-        source_fields = parse_qs(parse_uri.query)
-        cred_path = source_fields.get("credentials_path")
-        cred_base64 = source_fields.get("credentials_base64")
-
-        if not cred_path and not cred_base64:
+        import ingestr.src.google_analytics.helpers as helpers
+        result = helpers.parse_google_analytics_uri(uri)
+        credentials = result["credentials"]
+        property_id = result["property_id"][0]
```

Comment:
if you already have a helper function then that should return a proper response, which means if you'll anyway take just the first property id then just return one instead of a list.